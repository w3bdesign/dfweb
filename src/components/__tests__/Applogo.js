// https://www.gatsbyjs.org/docs/unit-testing/

import React from "react"
import renderer from "react-test-renderer"

import AppLogo from "../Header/Applogo.component"

describe("AppLogo", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<AppLogo />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})