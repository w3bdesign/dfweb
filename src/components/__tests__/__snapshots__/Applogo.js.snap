// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppLogo renders correctly 1`] = `
<div
  className="flex w-full font-extrabold text-white md:w-1/2 md:justify-start"
>
  <a
    className="text-white"
    href="/"
  />
</div>
`;
